### DISCUSSION TITLE
Add vote/created event to Webhooks

### DISCUSSION THREAD

**Discussion by limone-eth** [snapshot-labs/snapshot#2193] [Ideas]
My idea is to add webhook events also for votes, as right now the only events supported are related to proposals.

I actually really need something like this to avoid periodically polling Snapshot APIs to get votes for certain addresses, which is not an optimal solution. I believe this is something useful not only for building with Snapshot APIs but also for improving the usage of such API (on my use case it would save a lot of useless http calls towards it) reducing the overall traffic a lot.

**Comment by samwhosung** [comment]
@lim0n3 yes this is something we are working towards. There are some other things we need to solve before.

**Reply by limone-eth** [reply]
Thanks for the quick response, great that you already have that in plan. How far you think you are from implementing this (approximately)? Are we talking about weeks, months...?
